fixed: situation when facebook user doesn't have a primary email

adnkh opened the pull request.

dev-101 commented:
Hi adnkh,
you should never close PR untill it gets accepted (or rejected), otherwise it may be missed by devs.

adnkh commented:
Hi dev-101
I have fixed some thing in this code, however this is the new changes:
https://github.com/osclass/plugin-facebook/pull/5

The problem happens when some body trying to login using facebook, and he does not have a valid or primary E-mail in his facebook account, in your code you send hem to logout link, but Which occurs is the cookies stuck in the browser and the website doesn't open .

I added some lines to delete all cookies for the program.
